evaluator/builtin_time.go

```diff
@@ -390,6 +390,16 @@ func builtinCurrentTime(args []types.Datum, _ context.Context) (d types.Datum, e
 	return convertToDuration(d, fsp)
 }
 
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_utc-date
+func builtinUTCDate(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	year, month, day := time.Now().Date()
+	t := mysql.Time{
+		Time: time.Date(year, month, day, 0, 0, 0, 0, time.Local),
```

Comment:
Well, we should drop the hour/minute/second before convert date to int, but fix it in [ToNumber](https://github.com/pingcap/tidb/blob/master/mysql/time.go#L223), I dont think its a good idea.